src/intTest/java/com/box/sdk/BoxFileIT.java

```diff
@@ -225,6 +227,64 @@ public void uploadAndDownloadFileSucceeds() throws IOException {
 
     }
 
+    @Test
+    public void uploadAndDownloadFileUseZstdSucceeds() throws IOException {
```

Comment:
maybe change the name to downloadFileUseZstdSucceeds, actually upload is just precondition to download file and this name suggests that we upload file using zstd as well, which is not the case